app/src/pages/[resourceType]/[id].tsx

```diff
@@ -84,30 +83,83 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
     }
   }, [dataRequirements]);
 
-  const draftMutation = trpc.draft.createDraft.useMutation({
-    onSuccess: data => {
-      notifications.show({
-        title: `Draft ${jsonData.resourceType} Created!`,
-        message: `Draft ${jsonData.resourceType}/${jsonData.id} successfully created`,
-        icon: <CircleCheck />,
-        color: 'green'
-      });
-      router.push(`/authoring/${jsonData.resourceType}/${data.draftId}`);
-      ctx.draft.getDraftCounts.invalidate();
+  const successNotification = (
+    resourceType: string,
+    createdFromArtifact: boolean,
```

Comment:
I'm having a hard time thinking of a situation when `createdFromArtifact` will be `false` here since we are creating a draft of a specific artifact from this page. What are examples where `createdFromArtifact` will be false? Or is it only the case that this input will be false when we are on the authoring page (which I think has its own notification functions)?